cypress.json

```diff
@@ -3,5 +3,6 @@
   "projectId": "m45k1x",
   "reporterOptions": {
     "mochaFile": "junit.xml"
-  }
+  },
+  "chromeWebSecurity": false
```

Comment:
this is so that cypress doesn't freak out when we from spruce.com to evergreen.com

Reply:
I don't think this is necessary - cypress tests should be using localhost:9090 as the app server, and that's already configured to allow CORS from localhost:3000